dev/debug.css

```diff
@@ -1 +1 @@
-../css/debug.css
\ No newline at end of file
+../src/css/debug.css
```

Comment:
Nice one :)

For information, the `npm run test:watch` was failing because it could not load this css files (because it moved to `./src` dir)